.eslintrc

```diff
@@ -105,7 +105,9 @@
     "react/no-array-index-key": 0,
     "react/no-unescaped-entities": 0,
     "react/no-unused-prop-types": 0,
-    "react/require-default-props": 0
+    "react/require-default-props": 0,
+    "react-hooks/rules-of-hooks": "error",
+    "react-hooks/exhaustive-deps": "warn"
```

Comment:
not necessary for this but how do you feel about switching to using these strings across the whole eslintrc?

Reply:
We should probably do that.

Reply:
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/17714